app/models/member.rb

```diff
@@ -121,6 +121,12 @@ def can_update_namespace_with_group_link?(user, object_namespace)
       can_modify?(user, object_namespace)
     end
 
+    def can_submit_workflow?(user, object_namespace)
+      Member::AccessLevel.manageable.include?(
```

Comment:
The `manageable` access levels are only `maintainer` and `owner`. A user can submit a workflow with a minimum access level of `Analyst`. We will probably want to create another method in the `AccessLevel` class such as `workflow_submittable` or something that maybe sounds a bit better, which we can reuse

Reply:
Is it okay if I do what we did for can_export_data? ed16a029c2c9ee0cc099bf525a0edfc29f386376

Reply:
Yup that is definitely good! Thanks!